packages/default-theme/package.json

```diff
@@ -21,6 +21,7 @@
     }
   },
   "dependencies": {
+    "@babel/preset-env": "^7.8.2",
```

Comment:
why is this needed?

Reply:
It was probably caused obsolete node_modules.